services/cd-service/pkg/repository/transformer.go

```diff
@@ -120,14 +120,12 @@ func commitEventDir(fs billy.Filesystem, commit, eventId string) string {
 	return fs.Join(commitDirectory(fs, commit), "events", eventId)
 }
 
-func GetEnvironmentLocksCount(fs billy.Filesystem, env string) float64 {
-	envLocksCount := 0
-	envDir := environmentDirectory(fs, env)
-	locksDir := fs.Join(envDir, "locks")
-	if entries, _ := fs.ReadDir(locksDir); entries != nil {
-		envLocksCount += len(entries)
+func (s *State) GetEnvironmentLocksCount(ctx context.Context, env string) float64 {
+	locks, err := s.GetEnvironmentLocks(ctx, env)
+	if err != nil {
+		return 0
```

Comment:
please do not ignore errors.
Ideally, return the error from this function and handle it in `GaugeEnvLockMetric` by logging